code/modules/urist/modules/weather.dm

```diff
@@ -0,0 +1,53 @@
+//weather effects
+
+/obj/effect/weather
+	name = "weather"
+	icon = 'icons/urist/weather.dmi'
+	icon_state = "bsnow"
+
+	layer = 5
+	anchored = 1 //prevents weather from being /draggable/
+	mouse_opacity = 0 //doesn't need to be clickable and is less of an annoyance for players
+
+/obj/effect/weather/blowingsnow
+	name = "blowing snow"
+	icon_state = "bsnow"
+	var/safe = 0 //safe == not arctic; ugly, but better than changing the vars on the map
+
+/obj/effect/weather/blowingsnow/Crossed(O as mob)
+	..()
+	if(!safe)
+		if(istype(O, /mob/living/))
+			var/mob/living/M = O
+			if(COLD_RESISTANCE in M.mutations)
+				M << ("<span class='warning'> The cold wind feels surprisingly pleasant to you.</span>")
+			else
+				if(prob(85))
+					M.apply_damage(rand(3,5), BURN)
+					M << ("<span class='warning'> The cold wind tears at your skin!</span>")
+
+/obj/effect/weather/rain
+	name = "rain"
+	icon_state = "rain"
+	color = null //fully supports colors, the null works but is very subtle
+
+/obj/effect/weather/rain/Crossed(O as mob)
+	..()
+	if(istype(O, /mob/living/carbon/))
+		var/mob/living/carbon/M = O
+		M.ExtinguishMob()
```

Comment:
good thinking. should also put out hotspots, but that's something that can be added later, for now this is good.